### DISCUSSION TITLE
Welcome to amr-wind Discussions!

### DISCUSSION THREAD

**Discussion by sayerhs** [Exawind/amr-wind#369] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by AsimUdel** [comment]
Hello,
I try to run simulation for big domain (10 km x 5 km x 1.92 km ) to see the wake of wind farm (currently I am running without wind  turbines), but I am facing a specific error : "terminate called after throwing an instance of 'std::runtime_error'
 what():  MLMG failing so lets stop here" ,
has anyone got this error in the past ? 
I have attached the files:
1) input file 
[input_file.txt](https://github.com/Exawind/amr-wind/files/14764907/input_file.txt)
2) error output file 
[log_output_file.txt](https://github.com/Exawind/amr-wind/files/14764911/log_output_file.txt)
3) screenshot 
![error](https://github.com/Exawind/amr-wind/assets/156853074/588d5adc-10b0-41a2-9006-674013f1c99e)
thank you in advance
